examples/asr/experimental/conformer/confs/conformer_char.yaml

```diff
@@ -0,0 +1,160 @@
+name: &name "Conformer-char"
+
+model:
+  sample_rate: &sample_rate 16000
+  labels: &labels [" ", "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m",
+                   "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "'"]
+  log_predictions: true
+  load_weights_from_checkpoint: null
+  ctc_reduction: 'mean_batch'
+
+  train_ds:
+    manifest_filepath: ???
+    labels: *labels
+    sample_rate: 16000
+    batch_size: 16
+    trim_silence: false
+    max_duration: 16.7
+    min_duration: 0.1
+    shuffle: true
+    is_tarred: false
+    tarred_audio_filepaths: null
+    num_workers: 4
+    pin_memory: true
+
+  validation_ds:
+    manifest_filepath: ???
+    labels: *labels
+    sample_rate: 16000
+    batch_size: 16
+    shuffle: false
+    num_workers: 4
+    pin_memory: true
+
+  test_ds:
+    manifest_filepath: null
+    labels: *labels
+    sample_rate: 16000
+    batch_size: 16
+    shuffle: false
+    num_workers: 4
+    pin_memory: true
+
+  preprocessor:
+    _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
+    sample_rate: *sample_rate
+    normalize: "per_feature"
+    window_size: 0.025
+    window_stride: 0.01
+    window: "hann"
+    features: &n_mels 80
+    n_fft: 512
+    frame_splicing: 1
+    dither: 0.00001
+    pad_to: 16
+    stft_conv: false
+
+  spec_augment:
+    _target_: nemo.collections.asr.modules.SpectrogramAugmentation
+    # SpecAug params
+    freq_masks: 2 # set to zero to disable the SpecAug augmentation
+    time_masks: 2 # set to zero to disable the SpecAug augmentation
+    freq_width: 27
+    time_width: 100
+    # Cut-off params
+    rect_masks: 0 # set to zero to disable the cut-off augmentation
+    rect_time: 120
+    rect_freq: 50
+
+  encoder:
+    _target_: nemo.collections.asr.modules.ConformerEncoder
+    feat_in: *n_mels
+    feat_out: -1 # you may set it if you need different output size other than the default d_model
+    n_layers: 16
+    d_model: 256
+
+    # Sub-sampling params
+    subsampling: vggnet # vggnet or striding
+    subsampling_factor: 4 # must be a multiply of 2
```

Comment:
Should this be a multiple of 2 or 2^x? As in 4 means 2^4 = 16x total stride? Or 4 means 4x stride

Reply:
This should be power of 2 actually. would fix it.